Threading, nrel_main_idtf for sc_definitions, docs fixes

pavellos21 opened the pull request:
What is done:

* using threads in context loading
* nrel_main_idtf for sc_definitions
* some fixes in docs

pavellos21 committed:
perf(loader): add loading in threads

Commit:
docs(readme): some fixes

pavellos21 committed:
feat(templates): nrel_main_idtf for sc_definitions

pavellos21 committed:
Merge branch 'master' of https://github.com/Flombik/data2sc-parser

pavellos21 committed:
Merge branch 'threading'

MikhailSadovsky commented on src/loaders/WikiDataWithContextLoader.py:
```diff
@@ -1,62 +1,84 @@
+                            re.sub(r"'s?|\(|\)|,", '', self._page.data['title']))
+        loading_queue = Queue()
+
+        for _ in range(16):
```
Please move thread count to class constant

MikhailSadovsky commented on src/templates/entity.scs:
```diff
@@ -3,6 +3,8 @@
     (*
+		=> nrel_main_idtf:{% for lang, label in labels %}
+			{% if lang == 'ru' %}[Опр.({{label}})]{% else %}[Def.({{label}})]{% endif %} (* <-lang_{{lang}};; *);{% endfor %};
 	    <- sc_definition;;
```
Please define "sc_definition" before "nrel_main_idtf"

MikhailSadovsky commented on src/templates/relation.scs:
```diff
@@ -3,6 +3,8 @@
     (*
+		=> nrel_main_idtf:{% for lang, label in labels %}
+			{% if lang == 'ru' %}[Опр.({{label}})]{% else %}[Def.({{label}})]{% endif %} (* <-lang_{{lang}};; *);{% endfor %};
 	    <- sc_definition;;
```
Please define "sc_definition" before "nrel_main_idtf"

pavellos21 committed:
fix: some fixes

Threads number moved to class constant.
Now defines "sc_definition" before "nrel_main_idtf".
